x/ibc/simulation/genesis.go

```diff
@@ -57,4 +57,5 @@ func RandomizedGenState(simState *module.SimulationState) {
 
 	fmt.Printf("Selected randomly generated %s parameters:\n%s\n", host.ModuleName, codec.MustMarshalJSONIndent(simState.Cdc, ibcGenesis))
 	simState.GenState[host.ModuleName] = simState.Cdc.MustMarshalJSON(ibcGenesis)
+	fmt.Println(simState.GenState[host.ModuleName])
```

Comment:
is this line necessary? I don't see gen states printed in other simulation pkgs for other moudles

Reply:
> added in #6712 . @dauTT do you think it is safe for us to delete this test now or how should it be adjusted to account for the new GenesisState behaviour?

@colin-axner , I would suggest to adjust the test to take into account the new behaviour.

Reply:
Ahh think this was me debugging